integration-testing/run_tests.sh

```diff
@@ -11,4 +11,4 @@ export DEFAULT_IMAGE=rchain-integration-testing:$tag
 sed "s/rnode:latest/rnode:$tag/" Dockerfile |\
 	docker build -t $DEFAULT_IMAGE -f - .
 
-pipenv run py.test -v "$@"
+pipenv run py.test -v --mypy "$@"
```

Comment:
```suggestion
pipenv run py.test -v "$@"
```

It may be too early to enable it by default.  Let's use it locally first to get a sense of its strengths and weaknesses and then enable it for everybody if all goes well.